data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,401 @@ mission "Remnant: Face to Maw 2B"
+			`	"So no, I don't have any jobs 'just like the last one,' but I know our researchers would like another shipment. Are you interested?"`
+			choice
+				`	(Sign the affirmative.)`
+				`	(Decline.)`
+					decline
+			`	"Great. They would like you to pick up another 19 Keystones. Two differences this time: First, we would like you to pick them up from several different planets. And second, by the time you reach Hai space we will have made arrangements so the shipments will be already paid for and waiting for you. Pick up the first stones on Allhome, then the next set on Greenwater. I will have a list sent to you by the time you reach Greenwater."`
+			`	"That being said, if you happen to pick up any additional crystals, you can turn them into the logistics office. They will handle the distribution and compensate you accordingly."`
+			choice
+				`	"Wait, how will you get this all set up?"`
+				`	"Sounds good."`
+					accept
+			branch remnantuntrusted
+				"remnantuntrusted" >= 1 # This attribute is currently not used, but will be set by some of the "consequence" missions as a result of spying for the Deep.
+			`	Aeolus looks faintly amused "While you are certainly adventurous, you are not the only one to have traveled widely. We mapped the Hai systems long ago with cloaked ships, but never landed or contacted any of them. So we never knew anything more than what we could observe from orbit or from scanning ships. But you are known to them, and have a fairly public reputation. By using your transponder codes, we can place and pay for orders through hyperspace relays. All we need is a ship in position to feed the signal into the network.`
+				accept
+			label remnantuntrusted
+			`	Aeolus smiles mysteriously at you. "Let's just say that you are not the only resourceful person working for us. We will share more with you once you have demonstrated your loyalty."`
+				accept
+	on complete
+		dialog
+			`It takes a few minutes to contact the yardmaster, but once you do the Hai have the crates moved promptly from the warehouse to your bay doors. Hai and Humans seem accustomed to working together here.`
+
+
+
+mission "Remnant: Crystal Research 2"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from Greenwater, and then Allhome.`
+	landing
+	source "Greenwater"
+	destination "Allhome"
+	cargo "Keystones" 2
+	to offer
+		has "Remnant: Crystal Research 1: done"
+	on complete
+		payment 250000
+		dialog
+			`True to Aeolus' word, there are crates of Hai Quantum Keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message from Viminal has arrived. It is a notification that a payment of <payment> has been made to your account, with a note attached that simply states "on-assignment allowance."`
```

Comment:
It's been established that instantaneous communication relies on hyperspace relays, but Vimnal is not connected to Allhome via a direct hyperspace route, so an instant message from Vimnal shouldn't be possible. Maybe rephrase this to make it clear that the message is pre-recorded, or something like that.

Reply:
I was thinking a text message. Literally like how a credit card statement has "payee: amazon.com" or some such written next to the line. "payee: on-assignment allowance." I'm actually kind of hoping that the player notices that very discrepancy, and wonders why that is the case. (behind the scenes, I'm explaining it to myself as either 

a) the Remnant planted a device and/or code on the player's ship to activate and make the payment when conditions are met; or 
b) the Remnant are following the player in a cloaked ship and are transmitting the payment when they can tell the player has made the pickup.

Reply:
OK, I edited out the mention of viminal, and it now emphasizes that it doesn't mention where it is from. So now it is clear that this is not a normal message that, despite appearing on your commlink, is clearly irregular.

Reply:
LGTM!